### PR TITLE
[python] Makes gpt-j model use triton mode

### PR DESCRIPTION
**Pull request by frankfliu**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: tests/integration/llm/fastertransformer-model.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def load_model(properties):
     tensor_parallel_degree = properties["tensor_parallel_degree"]
     pipeline_parallel_degree = 1  # TODO: add tests for pp_degree > 1
     model_id = properties.get('model_id') or properties.get('model_dir')
-    use_triton = properties.get("use_triton", False)
+    use_triton = properties.get("engine") != "FasterTransformer"
```
Comment (lanking520) [inline tests/integration/llm/fastertransformer-model.py:12]: We passed in DeepSpeed in some tests, this check doesn't work

File: engines/python/setup/djl_python/fastertransformer.py [hunk 2]
```diff
@@ -120,18 +125,25 @@ def inference(self, inputs: Input):
                         )
 
             parameters = self.param_mapper(parameters)
-            if self.is_t5:
-                result = self.model.pipeline_generate(input_data, **parameters)
+            max_length = parameters.pop("max_length", 50)
+            output_len = parameters.pop("max_seq_len", max_length)
+            if self.use_triton:
+                output_length = [output_len] * len(input_data)
+                result = self.model.pipeline_generate(input_data,
```
Comment (lanking520) [inline engines/python/setup/djl_python/fastertransformer.py:132]: append with **parameters
